iir/Common.h

```diff
@@ -46,10 +46,12 @@
 
 // This exports the classes/structures to the windows DLL
 #ifdef _WIN32
-#define DllExport   __declspec( dllexport )
-#define _CRT_SECURE_NO_WARNINGS
+#  define DllExport   __declspec( dllexport )
+#  ifndef _CRT_SECURE_NO_WARNINGS
+#    define _CRT_SECURE_NO_WARNINGS
+#  endif
```

Comment:
Applying similar preprocessor indention as used a couple lines above with the pragma.